src/state/opamEnv.ml

```diff
@@ -371,37 +373,40 @@ let init_sh        = "init.sh"
 let init_zsh       = "init.zsh"
 let init_csh       = "init.csh"
 let init_fish      = "init.fish"
+let opam_env_sh    =
+  "_opam_env_hook() {\n\
```

Comment:
the `\` should be below the opening `"` for correct alignment. `ocp-indent` should do it that way automatically... ?